docs/compatibility.md

```diff
@@ -0,0 +1,77 @@
+# StableHLO Compatibility
+
+StableHLO is a backward compatible ML compute opset inspired by HLO/MHLO.
+This document explains the kind and the extent of the compatibility guarantees
+that StableHLO provides, based on the process established in
+[the compatibility RFC](../rfcs/20220912-compatibility.md).
+
+## Versions
+
+The current version of StableHLO is **0.9.0**.
```

Comment:
This is ok for now, but will go stale on updates, unless we intend to keep updating this document. Once we have a releases page we can link to it.

Reply:
Yeah, I was thinking to write up release_checklist.md which would include a process for creating new releases, including updating this document. On a separate note, yes, linking to the releases page is something that's on my todo list after we tag our first release.